wiki/Legal/Terms/id.md

```diff
@@ -0,0 +1,118 @@
+ATAS OPSI DARI osu!, KEWAJIBAN KAMI ATAS PELANGGARAN KONDISI DAN JAMINAN YANG TERSIRAT ADALAH TERBATAS, HANYA SEJAUH YANG DIIZINKAN OLEH HUKUM SAJA, JIKA PELANGGARAN TERSEBUT TERKAIT DENGAN BARANG, (I) PENGGANTIAN BARANG ATAU PENYEDIAAN BARANG YANG SETARA, (II) PERBAIKAN BARANG, (III) PEMBAYARAN BIAYA SAAT MENGGANTI BARANG ATAU MENDAPATKAN BARANG YANG SETARA, ATAU (IV) BIAYA PERBAIKAN BARANG DAN, JIKA PELANGGARAN TERKAIT DENGAN LAYANAN, (I) PENYEDIAAN LAYANAN KEMBALI; ATAU (II) PEMBAYARAN BIAYA DARI PENYEDIAAN LAYANAN KEMBALI.
+
+ANDA SECARA KHUSUS MENGAKUI BAHWA osu! TIDAK AKAN BERTANGGUNG JAWAB ATAS KIRIMAN PENGGUNA ATAU PERILAKU MEMFITNAH, MENYERANG, ATAU ILEGAL DARI PIHAK KETIGA, DAN RISIKO KECELAKAAN DAN KERUSAKAN TERSEBUT DITANGGUNG OLEH ANDA.
+
+Situs web ini dikendalikan dan ditawarkan oleh osu! dari fasilitasnya di Amerika Serikat.
+osu! tidak membuat pernyataan bahwa Situs Web osu! sesuai atau tersedia untuk digunakan di lokasi lain.
+Mereka yang mengakses atau menggunakan Situs Web osu! dari yuridiksi lain dan melakukannya atas kemauan mereka sendiri, bertanggung jawab untuk mematuhi hukum setempat.
+
+## GANTI RUGI
+
+Anda setuju untuk membela, mengganti kerugian dan membebaskan osu!, perusahaan induknya, pejabat, direktur, karyawan dan agennya, dari dan terhadap setiap dan semua klaim, kerusakan, kewajiban, kerugian, tanggung jawab, biaya atau hutang, dan pengeluaran (termasuk biaya pengacara tetapi tidak terbatas) yang timbul dari: (i) penggunaan dan akses Anda terhadap Situs Web osu!; (ii) pelanggaran Anda terhadap persyaratan apa pun dari Ketentuan Layanan ini; (iii) pelanggaran Anda terhadap pihak ketiga mana pun, termasuk hak cipta, properti, atau hak privasi yang tidak terbatas; atau (iv) setiap klaim bahwa salah satu Kiriman Pengguna menyebabkan kerusakan pada pihak ketiga.
+Kewajiban pembelaan dan ganti rugi ini akan tetap berlaku pada Ketentuan Layanan dan penggunaan Anda atas Situs Web osu!.
+
+## PENGGUNAAN ANDA TERHADAP KONTEN DI SITUS WEB
+
+Anda memahami bahwa saat menggunakan osu!, Anda akan dihadapkan pada Kiriman Pengguna dari berbagai sumber, dan osu! tidak bertanggung jawab atas keakuratan, kegunaan, keamanan, atau hak kekayaan intelektual dari atau terkait dengan Kiriman Pengguna tersebut.
+Anda lebih memahami dan mengakui bahwa Anda mungkin Kiriman Pengguna yang tidak akurat, menyinggung, tidak senonoh, atau tidak menyenangkan, dan Anda setuju untuk mengesampingkan, dan dengan ini mengesampingkan, segala hak atau upaya hukum atau kesetaraan yang Anda miliki atau mungkin Anda miliki terhadap osu! sehubungan dengan hal tersebut, dan setuju untuk mengganti kerugian dan menahan osu!, Pemilik/Operatornya, afiliasi, dan/atau pemberi lisensinya, sejauh yang diizinkan oleh hukum tentang semua hal yang terkait dengan penggunaan Anda atas situs ini.
```

Comment:
```suggestion
Anda lebih memahami dan mengakui bahwa mungkin terdapat Kiriman Pengguna yang tidak akurat, menyinggung, tidak senonoh, atau tidak menyenangkan, dan Anda setuju untuk mengesampingkan, dan dengan ini mengesampingkan, segala hak atau upaya hukum atau kesetaraan yang Anda miliki atau mungkin Anda miliki terhadap osu! sehubungan dengan hal tersebut, dan setuju untuk mengganti kerugian dan menahan osu!, Pemilik/Operatornya, afiliasi, dan/atau pemberi lisensinya, sejauh yang diizinkan oleh hukum tentang semua hal yang terkait dengan penggunaan Anda atas situs ini.
```
i think this one is the correct one

Reply:
ada terjemahan yang kurang disitu kayanya, diganti jadi `Anda lebih memahami dan mengakui bahwa Anda mungkin dapat terlihat pada Kiriman Pengguna yang tidak akurat` ya.